AmplifyPlugins/Core/AWSPluginsCore/ServiceConfiguration/AmplifyAWSServiceConfiguration.swift

```diff
@@ -9,15 +9,16 @@ import Foundation
 import AWSCore
 
 public class AmplifyAWSServiceConfiguration: AWSServiceConfiguration {
+    private static let version = "1.1.0"
```

Comment:
have you verified that this new version declaration works with the fastlane version bump script?

Reply:
Yes tested this with help of @iartemiev . Ran `bundle exec fastlane unstable` and verified that the version is updated.